polygon-digital-carbon/subgraph.yaml

```diff
@@ -353,20 +353,11 @@ dataSources:
       abis:
         - name: CarbonmarkCreditTokenFactory
           file: ../lib/abis/CarbonmarkCreditTokenFactory.json
-        - name: C3ProjectToken
+        - name: CarbonmarkCreditToken
           file: ../lib/abis/CarbonmarkCreditToken.json
-        - name: ERC20
-          file: ../lib/abis/ERC20.json
-        # TODO: determine whether to include
-        # - name: CarbonmarkCertificateNFT
-        #   file: ../lib/abis/CarbonmarkCertificateNFT.json
       eventHandlers:
-        # TODO: implement this handler
-        - event: Issued(string,uint256,address,address)
+        - event: Issued(indexed string,uint256,indexed address,indexed address)
           handler: handleNewCarbonmarkCredit
-        # TODO: determine if we need a handler for cancellation event
```

Comment:
Why are we removing this TODO? Should be left in for future reference unless this PR adds cancellation handling logic

Reply:
Done in 7297431